.github/workflows/build.yml

```diff
@@ -0,0 +1,26 @@
+on: push
+jobs:
+  build:
+    runs-on: macos-latest
+    env:
+      LDFLAGS: -L/usr/local/opt/openblas/lib
+      CPPFLAGS: -I/usr/local/opt/openblas/include
+      MAKEFLAGS: "-j $(sysctl -n hw.logicalcpu)"
+    steps:
+    - uses: actions/checkout@v2
+    - run: brew upgrade
+    - run: brew install openblas swig boost python3 tbb nlopt cminpack ceres-solver bison flex hdf5 ipopt r
+    - run: cd $HOME/work/openturns && git clone https://github.com/yixuan/spectra.git spectra
+    - run: cd $HOME/work/openturns/spectra && git checkout v0.9.0
+    - run: cd $HOME/work/openturns/spectra && cmake -DBUILD_TESTS=TRUE -DCMAKE_INSTALL_PREFIX=$HOME/ .
```

Comment:
seems you can write a long command in a single run instruction, with | and \ to skip lines if needed

Reply:
Done